animeshon/knowledge/v1alpha1/knowledge.proto

```diff
@@ -18,66 +19,123 @@ service Knowledge {
 
   rpc CreateContribution(CreateContributionRequest) returns (Contribution) {
     option (google.api.http) = {
-      post: "/v1alpha1/contributions"
+      post: "/v1alpha1/{parent=users/*}/contributions"
       body: "*"
     };
   }
 
   rpc ListContributions(ListContributionsRequest) returns (ListContributionsResponse) {
     option (google.api.http) = {
-      get: "/v1alpha1/contributions"
+      get: "/v1alpha1/{parent=users/*}/contributions"
     };
   }
 
   rpc GetContribution(GetContributionRequest) returns (Contribution) {
     option (google.api.http) = {
-      get: "/v1alpha1/{name=contributions/*}"
+      get: "/v1alpha1/{name=users/*/contributions/*}"
+    };
+  }
+
+  rpc GetContributionChanges(GetContributionChangesRequest) returns (ContributionChanges) {
+    option (google.api.http) = {
+    get: "/v1alpha1/{name=users/*/contributions/*/changes}"
     };
   }
 
   rpc ApproveContribution(ApproveContributionRequest) returns (Contribution) {
     option (google.api.http) = {
-      post: "/v1alpha1/{name=contributions/*}:approve"
+      post: "/v1alpha1/{name=users/*/contributions/*}:approve"
+      body: "*"
+    };
+  }
+
+  rpc ReviewContribution(ReviewContributionRequest) returns (Contribution) {
```

Comment:
Rename method to `UpdateContribution` with the format described in https://google.aip.dev/134.